scripts/build/utils/build-theme.ts

```diff
@@ -0,0 +1,31 @@
+import * as fs from "fs";
```

Comment:
To be consistent with
https://github.com/cadamsdev/vscode-jetbrains-icon-theme/pull/103/files#diff-7d51ff9d2b2fe28ba3bad3e3a41dddf848d80a6a7e5a184d16317a00b980733dR3

```suggestion
import * as fs from "node:fs";
```